GDJS/Runtime/scenestack.ts

```diff
@@ -70,6 +70,44 @@ namespace gdjs {
         } else {
           logger.error('Unrecognized change in scene stack: ' + request);
         }
+
+        const allSyncData: GameSaveState = JSON.parse(
```

Comment:
You do this getItem and parse at every frame, this seems wasteful for the CPU because it's then immediately discarded.